tests/test_project.py

```diff
@@ -67,30 +66,6 @@
 WINDOWS = sys.platform == "win32"
```

Comment:
Should this be aligned with https://github.com/glotzerlab/signac/pull/850/files#diff-2536a92baaf2e0ef0a296bc63eb34b07ed102cf82c7f6d50f13a68b862d138c6R31 ?

Reply:
*sigh* Unfortunately no. We're really testing for two different functionalities. The multithreaded failures occur on both Windows and Cygwin, but the linked view failures occur only on Windows and _not_ Cygwin.

Reply:
The right solution is probably to refactor both checks to describe exactly what they're checking, and not just "Windows." It's really two different sets of functionality.